### PR TITLE
Websocket integration - events notifier

### PR DESCRIPTION
**Pull request by ssd04**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [x] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/notifier.md [hunk 1]
```diff
@@ -48,13 +48,70 @@ In the figure above:
...
-In the observer node's configuration directory, `external.toml` config file can be configured
-to enable notifier connector. The config file can be found 
+In the observer node's configuration directory, the `external.toml` config file can be configured
+to enable events notifier connector via WebSocket or via HTTP integrations. The config file can be found 
 [here](https://github.com/multiversx/mx-chain-go/blob/master/cmd/node/config/external.toml).
 
-The supported config variables are as follows:
+[comment]: # (mx-context-auto)
+
+#### WebSocket Integration
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/notifier.md:60]: 4 `#` won't be included in the right side's "table on content". Can remain as it is, tho

File: docs/sdk-and-tools/notifier.md [hunk 2]
```diff
@@ -30,11 +30,11 @@ Set up at least one observer for each shard in order to handle all the events in
...
 In the figure above:
-- The observer nodes will push block events to Notifier instance, via HTTP POST requests. There are several endpoints for this:
-    - `/events/push` (POST) -> it will handle all events for each round
-    - `/events/revert` (POST) -> if there is a reverted block, the event will be
+- The observer nodes will push block events to Notifier instance, via WebSocket or HTTP requests. There are several endpoints/topics for this:
+    - `block events` -> it will handle all events for each round
+    - `revert events` -> if there is a reverted block, the event will be
       pushed on this route
-    - `/events/finalized` (POST) -> when the block has been finalized, the events
+    - `finalized events` -> when the block has been finalized, the events
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/notifier.md:37]: replace `Websockets` with `Websocket` on L40 and other occurrences

File: docs/sdk-and-tools/notifier.md [hunk 3]
```diff
@@ -48,13 +48,70 @@ In the figure above:
...
+In the observer node's configuration directory, the `external.toml` config file can be configured
+to enable events notifier connector via WebSocket or via HTTP integrations. The config file can be found 
 [here](https://github.com/multiversx/mx-chain-go/blob/master/cmd/node/config/external.toml).
 
-The supported config variables are as follows:
+[comment]: # (mx-context-auto)
+
+#### WebSocket Integration
+
+This WebSocket integration is a generic one, and can be used for multiple outport driver integrations.
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/notifier.md:62]: This -> The

File: docs/sdk-and-tools/notifier.md [hunk 4]
```diff
@@ -48,13 +48,70 @@ In the figure above:
...
+    URL = "127.0.0.1:22111"
+
+    # After a message will be sent it will wait for an ack message if this flag is enabled
+    WithAcknowledge = true
+
+    # The duration in seconds to wait for an acknowledgment message, after this time passes an error will be returned
+    AcknowledgeTimeoutInSec = 60
+
+    # Currently, only "json" is supported. In the future, "gogo protobuf" could also be supported
+    MarshallerType = "gogo protobug"
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/notifier.md:87]: protobuf*

File: docs/sdk-and-tools/notifier.md (hunk 2 above)
Comment (ssd04) [reply docs/sdk-and-tools/notifier.md:37]: changed

**Review by bogdan-rosianu — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**